src/backend/distributed/sql/udfs/citus_prepare_pg_upgrade/9.5-2.sql

```diff
@@ -0,0 +1,59 @@
+CREATE OR REPLACE FUNCTION pg_catalog.citus_prepare_pg_upgrade()
+    RETURNS void
+    LANGUAGE plpgsql
+    SET search_path = pg_catalog
+    AS $cppu$
+BEGIN
+    --
+    -- Drop existing backup tables
+    --
+    DROP TABLE IF EXISTS public.pg_dist_partition;
```

Comment:
Why do we have these DROPs here but not in 9.4-2?

Reply:
oh, wow! good catch!

Reply:
I guess the drops were introduced in 9.5: https://github.com/citusdata/citus/commit/2a6c867cb023d183fe9a27700081a71ce7e99bde

Reply:
yeah seems like using DROPs would be better on 9.4-2 as well for idempotency, but that seems a bit irrelevant for this PR